### PR TITLE
radio-group: Check coverage tests

### LINKED ISSUE #588: radio-group: Check coverage tests

### PR DESCRIPTION
**Pull request by DimaChervan**
Close #588

### CODE REVIEW

**Comment by sipayRT** [comment]
@tadatuta

**Comment by tadatuta** [comment]
LGTM

File: common.blocks/radio-group/radio-group.spec.js [hunk 1]
```diff
@@ -61,12 +61,23 @@ describe('radio-group', function() {
         });
 
         it('should not set wrong value', function() {
-            var options = radioGroup.getRadios(),
-                spy = sinon.spy();
+            radioGroup.setVal('val44');
```
Comment (gela-d) [inline common.blocks/radio-group/radio-group.spec.js:64]: It should be an array

Comment (gela-d) [inline common.blocks/radio-group/radio-group.spec.js:64]: Oh wait, in radio this is String, sorry :)

**Comment by dfilatov** [comment]
lgtm

**Pull request merged**